english/mirror/enter-mirror.md

```diff
@@ -5,3 +5,5 @@ It's not a mirror, it's a door to a marshmellow room that looks identical to the
 [Go back to the previous room](../marshmallow.md)
 
 [Look for a toilet](../eating-walls/find-a-toilet/find-a-toilet.md)
+
+[quit your job](../quit-yourjob/quit-your-job.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.